buildSrc/src/main/kotlin/com/datadog/gradle/config/FlavorBuildConfig.kt

```diff
@@ -65,4 +64,9 @@ fun configureFlavorForSampleApp(flavor: ProductFlavor, rootDir: File) {
         "DD_INTERNAL_MONITORING_CLIENT_TOKEN",
         "\"${config.internalMonitoringToken}\""
     )
+    flavor.buildConfigField(
+            "String",
+            "DD_SITE_NAME",
+            "\"${flavor.name.toUpperCase()}\""
```

Comment:
very nit: usage of `toUpperCase` without locale is discouraged - it may lead to unexpected results on some user locales.